server/game/core/Player.js

```diff
@@ -288,16 +288,33 @@ class Player extends GameObject {
     }
 
     /**
-     * Returns if a card is in play (unots, upgrades, provinces, holdings) that has the passed trait
+     * Returns if a card is in play (units, upgrades, provinces, holdings) that has the passed trait
      * @param {string} trait
-     * @returns {boolean} true/false if the trait is in pay
+     * @param {any} ignoreUnit
+     * @returns {boolean} true/false if the trait is in play
      */
-    isTraitInPlay(trait) {
+    isTraitInPlay(trait, ignoreUnit = undefined) {
```

Comment:
nit: I think `null` and `undefined` are mostly interchangeable in javascript but I think in this situation it's more conventional to use `null`. Same for `isAspectInPlay`